### ISSUE TITLE
Script is running without response

### ISSUE THREAD

**Issue by KevinChngJY** [t-wolfeadam/Alpyne#9]
When i'm trying to run the sample script, the script is running very long without response.
I open the alpyne.log, seem that the WADL feature is disabled. How do I enable this?

[2021-11-30 15:09:40] INFO    @ com.anylogic.alpyne.AlpyneServer main 
                              | [PID=5336] Initializing @ 'C:\Users\kchng2\AppData\Local\Temp\alpyne_63169_z2y1xnrp' with arguments: {path=., port=51150, dev_mode=false}
[2021-11-30 15:09:40] FINER   @ ModelManager.<init>
                              | ENTRY {0}	.
[2021-11-30 15:09:40] FINER   @ BonsaiBrainLearningPlatform.<init>
                              | ENTRY {0}	{}
[2021-11-30 15:09:40] INFO    @ Version.load
                              | Loading from experiment
[2021-11-30 15:09:40] INFO    @ Version.load
                              | Inputs      = [(INTEGER)mode=null, (DOUBLE)interarrivalSeconds=null, (BOOLEAN)applyMultipliers=null, (BOOLEAN)verbose=null, (DOUBLE){START_TIME}=0.0, (DATE_TIME){START_DATE}=Wed Sep 29 00:00:00 SGT 2021, (DOUBLE){STOP_TIME}=Infinity, (DATE_TIME){STOP_DATE}=null, (LONG){RANDOM_SEED}=null]
                                Outputs     = [(DOUBLE)interarrivalSeconds=null, (HISTOGRAM_DATA)delayDistribution=null, (DATA_SET)processSizeDS=null, (INTEGER)agentsInLimbo=null, (INTEGER)agentsProcessed=null]
                                Config      = [(DOUBLE)interarrival_seconds=null, (BOOLEAN)apply_multipliers=null, (BOOLEAN)logging=null, (DOUBLE){START_TIME}=0.0, (DATE_TIME){START_DATE}=Wed Sep 29 00:00:00 SGT 2021, (DOUBLE){STOP_TIME}=Infinity, (DATE_TIME){STOP_DATE}=null, (LONG){RANDOM_SEED}=null]
                                Observation = [(DOUBLE)agent_multiplier=null, (DOUBLE_ARRAY)times_remaining=null]
                                Action      = [(DOUBLE)process_time=null]
[2021-11-30 15:09:41] WARNING @ org.glassfish.jersey.server.wadl.WadlFeature configure 
                              | JAX-B API not found . WADL feature is disabled.
[2021-11-30 15:09:41] INFO    @ org.glassfish.grizzly.http.server.NetworkListener start 
                              | Started listener bound to [localhost:51150]
[2021-11-30 15:09:41] INFO    @ org.glassfish.grizzly.http.server.HttpServer start 
                              | [HttpServer] Started.

**Comment by KevinChngJY** [comment]
Somehow it is solved by java jdk 17, i'm not really sure that this is the solution, because i made many changes and out of sudden that the script is running well.

**Comment by t-wolfeadam** [comment]
I made my comment on your other issue about JDK 11 before I saw this, but that definitely may have been related. AnyLogic uses JDK11 so trying to run it with an earlier version of Java is likely to cause issues.

Also, don't worry about the WADL feature. That's a warning from the HTTP library I'm using but it doesn't have an effect on the logic. I'm planning on switching over to another library, so that should be resolved then.
